gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/amqp/task/TaskHandlerException.java

```diff
@@ -0,0 +1,11 @@
+package uk.nhs.adaptors.pss.translator.amqp.task;
+
+public class TaskHandlerException extends RuntimeException {
```

Comment:
what about getting rid of that class completely? maybe we could add it when we need it?